backend-go/internal/middleware/jwt_test.go

```
@@ -1,7 +1,8 @@
 package middleware
 
 import (
-	"fmt"
+	"encoding/json"
+	"github.com/yorkie-team/codepair/backend/api/codepair/v1/models"
```

golangci-lint (1.62.2)

File is not `goimports`-ed with -local github.com/yorkie-team/codepair

(goimports)